server/controllers/finance/accounts/import.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Account Import Module
+ *
+ * This module is responsible of handling the import of accounts
+ * and related stock quantities
+ */
+const Q = require('q');
+const path = require('path');
+
+const util = require('../../../lib/util');
+const db = require('../../../lib/db');
+const BadRequest = require('../../../lib/errors/BadRequest');
+
+const IMPORT_DEFAULT_OHADA_ACCOUNTS = 1;
+const IMPORT_CUSTOM_OHADA_ACCOUNTS = 2;
+const IMPORT_OTHER_ACCOUNTS = 3;
+
+exports.downloadTemplate = downloadTemplate;
+exports.importAccounts = importAccounts;
+
+/**
+ * @method downloadTemplate
+ *
+ * @description send to the client the template file for account import
+*/
+function downloadTemplate(req, res, next) {
+  try {
+    const file = path.join(__dirname, '../../../resources/templates/import-account-template.csv');
+    res.download(file);
+  } catch (error) {
+    next(error);
+  }
+}
+
+/**
+ * @method importAccounts
+ *
+ * @description this method allow to do an import of accounts
+ */
+function importAccounts(req, res, next) {
+  const params = util.convertStringToNumber(req.query);
+
+  if (params.option !== IMPORT_DEFAULT_OHADA_ACCOUNTS && (!req.files || req.files.length === 0)) {
+    next(new BadRequest('Something broke', 'ERRORS.EVERYTHING_BAD'));
+    return;
+  }
+
+  const file = req.files[0];
+  const dbPromises = [];
+
+  if (params.option === IMPORT_DEFAULT_OHADA_ACCOUNTS || params.option === IMPORT_CUSTOM_OHADA_ACCOUNTS) {
+    const basicAccountFile = path.join(__dirname, '../../../resources/templates/ohada-main-accounts.csv');
+    dbPromises.push(importAccountFromFile(basicAccountFile, req.session.enterprise.id, params.option));
+  }
+
+  if (params.option === IMPORT_CUSTOM_OHADA_ACCOUNTS || params.option === IMPORT_OTHER_ACCOUNTS) {
+    dbPromises.push(importAccountFromFile(file.path, req.session.enterprise.id, params.option));
+  }
+
+  Q.all(dbPromises)
+    .then(() => res.sendStatus(200))
```

Comment:
Since this is CREATE-ing accounts via a POST request, the best response code here is 201.